src/packages.gleam

```diff
@@ -20,6 +21,8 @@ const usage = "Usage:
 "
 
 pub fn main() {
+  let _ = dotenv.load()
```

Comment:
I currently have `glenvy` set as a `dev-dependency`, but am loading it in `main` (which will run in production).

I'm guessing this won't work?

Is there a directive we can use to only run certain code in development, or should we just promote `glenvy` to a regular dependency?

Reply:
I ended up promoting it to a regular dependency after some discussion in Discord.